specs/plugin-extensibility.md

```diff
@@ -493,13 +491,21 @@ This allows Notary v2 implementations to perform the same logic themselves, if t
     { 
        "contentType" : "application/vnd.cncf.notary.payload.v1+json",
        // One of notary.default.x509 or notary.signingAuthority.x509
-       "signingScheme" : "notary.default.x509",
+       "signingScheme" : "notary.default.x509" | "notary.signingAuthority.x509",
        // Value is always RFC 3339 formatted date time string
        "expiry": "2022-10-06T07:01:20Z", 
+       // if signingScheme is notary.signingAuthority.x509.
+       "authenticSigningTime": "2022-04-06T07:01:20Z",
+       // Name of the verification plugin
+       "verificationPlugin": "com.example.nv2plugin",
```

Comment:
`verificationPlugin` and `verificationPluginMinVersion`, which are mentioned in [JWS ](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md), and a general description in [signature format](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md#extended-attributes-for-notation-plugins), but not in [COSE](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md). Will this be a problem? @shizhMSFT  do you have any comments?

Reply:
It is supported in COSE. We should update the COSE doc accordingly.

Reply:
Opened issue to track this https://github.com/notaryproject/notaryproject/issues/232